package.json

```diff
@@ -69,5 +69,5 @@
     "uglify-js": "^2.4.14",
     "wd": "^0.3.8"
   },
-  "version": "2.3.1"
+  "version": "2.3.2"
```

Comment:
Whoops, this change is actually in master pre-branch, but I hadn't pushed master up to date.  Have now, but I think Github will continue to show this change on the PR.